### DISCUSSION TITLE
Wrong Markdown transformation of & character

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#111] [Report a bug] [answered]
```
 **Text & Text** 
```

is interpreted into

```
<strong>Text &amp; Text</strong>
```

I did not find a way, to get a "strong &" sign.

**Comment by markseuffert** [comment]
Can you try something out for me? Please switch off the safe mode on your server… and see if it makes any differences.

**Comment by markseuffert** [comment]
Here's what I found out. The handling of HTML entities was wrong when the safe mode was switched on. This is a problem in the Markdown library we are using, see michelf/php-markdown#173. I updated the Markdown plugin, which now should work closer to the Markdown specification: [If you use an ampersand as part of an HTML entity, it remains unchanged](http://daringfireball.net/projects/markdown/syntax#autoescape).

**Comment by markseuffert** [comment, marked as answer]
Should be working now.
